MapboxMobileEvents/MMEBackgroundLocationServiceTimeoutHandler.h

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+
+@class MMEBackgroundLocationServiceTimeoutHandler;
+@protocol MMEUIApplicationWrapper;
+
+@protocol MMEBackgroundLocationServiceTimeoutHandlerDelegate
```

Comment:
I hate this protocol name 😢 - any suggestions @rclee @fabian-guerra. Maybe replace `Handler` with just one of `Protocol`, `Delegate`, `Handler`?

Reply:
`MMEBackgroundLocationServiceTimeoutDelegate` works for me.